springfox-spring-web/src/main/java/springfox/documentation/spring/web/WebMvcRequestHandler.java

```diff
@@ -114,6 +116,11 @@ public String getName() {
   }
 
   @Override
+  public List<Annotation> getAnnotations() {
```

Comment:
Same here... I feel like a plugin should request only annotations it knows about. This is very open interface in my opinion.

Reply:
Agreed, last commit restored that.